README.md

```diff
@@ -27,6 +27,8 @@ Copies the local player's position and rotation onto the GameObject with the Udo
 
 Very messy script, check source code to see if it could be useful to you. Main features being `DateTime.Now.ToString()` formatting and `formatCustom` will replace `%playerLocal%` with the local player's display name.
 
+**For being able to use `forceAscii` feature, requires assigning ``Unidecoder`` object through a reference!**
```

Comment:
Something like this seems easier to understand.
> ** `forceAscii` requires a game object containing `Unidecoder.cs` assigned to the `unidecoder` variable.

Reply:
Changed